samples/omp_svr/mynewt/src/main.c

```diff
@@ -21,9 +21,9 @@
 #include <bsp/bsp.h>
 #include <hal/hal_gpio.h>
 #include <hal/hal_flash.h>
-#include <console/console.h>
+#include <zephyr/console/console.h>
```

Comment:
Please do not change this. It’s the mynewt specific sample.

Reply:
oops migration script runs for all files, assuming they're a Zephyr project. thanks for spotting